cmd/notify-mailer/main.go

```diff
@@ -108,119 +109,141 @@ func (m *mailer) run() error {
 		return err
 	}
 
-	m.log.Infof("Resolving %d destination addresses", len(m.destinations))
 	addressesToRecipients, err := m.resolveEmailAddresses()
 	if err != nil {
 		return err
 	}
-	if len(addressesToRecipients) == 0 {
-		return fmt.Errorf("zero recipients after looking up addresses?")
+
+	totalAddresses := len(addressesToRecipients)
+	if totalAddresses == 0 {
+		return errors.New("0 recipients remained after resolving addresses")
 	}
-	m.log.Infof("Resolved destination addresses. %d accounts became %d addresses.",
-		len(m.destinations), len(addressesToRecipients))
-	var biggest int
-	var biggestAddress string
-	for k, v := range addressesToRecipients {
-		if len(v) > biggest {
-			biggest = len(v)
-			biggestAddress = k
+
+	m.log.Infof(
+		"%d recipients were resolved to %d addresses",
+		len(m.recipients), totalAddresses)
+
+	var mostRecipientsLength int
+	var mostRecipientsAddress string
+	for emailAddress, recipients := range addressesToRecipients {
+		if len(recipients) > mostRecipientsLength {
+			mostRecipientsLength = len(recipients)
+			mostRecipientsAddress = emailAddress
 		}
 	}
-	m.log.Infof("Most frequent address %q had %d associated lines", biggestAddress, biggest)
+	m.log.Infof(
+		"The address with the most recipients: %q had: %d associated recipients",
```

Comment:
Related to hyper-nit above: reading this takes me a minute to parse. Is the key preceding the first colon "The address with the most recipients" (in which case I expect the value to be an address) or is it "recipients" (in which case I expect the value to be an integer)? I figured out that it is the former, but the formatting made that difficult to arrive at intuitively.